Replicator/IncomingRev+Blobs.cc

```diff
@@ -76,7 +76,14 @@ namespace litecore { namespace repl {
                         auto err = progress.reply->getError();
                         logError("Got error response: %.*s %d '%.*s'",
                                  SPLAT(err.domain), err.code, SPLAT(err.message));
-                        blobGotError(blipToC4Error(err));
+                        // c.f. CBL-4898
+                        std::optional<Error> cblErr;
+                        string errbuf;
+                        if (err.domain == "HTTP"_sl && err.code == 403) {
+                            errbuf = format("Blob in \"%.*s\" with digest \"%s\" is not in attachments.", SPLAT(_blob->docID), _blob->key.digestString().c_str());
```

Comment:
I think that since this message is meant to help the end user, it could be worded much better.  I propose:

"There is no content for the blob with digest %s in the attachments for document %s"

Reply:
Super! Will do it